docs/actions.md

```diff
@@ -567,16 +570,16 @@ The action may be invoked as any other OpenWhisk action.
   }
   ```
 
-If you update the Docker image, run `buildAndPush.sh` to refresh the image on Docker Hub, then run `wsk action update` to make the system to fetch the new image. New invocations will start using the new image and not a warm image with the old code.
-
+  To update the Docker image, run `buildAndPush.sh` to refresh the image on Docker Hub, then run `wsk action update` to make the system to fetch the new image. New invocations will start using the new image and not a warm image with the old code.
```

Comment:
@rabbah How about something like this:

To update the Docker action, run buildAndPush.sh to refresh the image on Docker Hub, this will allow next time the system pulls your Docker image to run the new code for your action. If there are no warm containers any new invocations will use the new Docker image. Take into account that if there is a warm container using a previous version of your Docker Image it will continue to use this unless you run `wsk action update`, this will indicate to the system that for any new invocations force a `docekr pull` resulting on pulling a your new Docker image.